assets/js/item_details.js

```diff
@@ -53,6 +59,29 @@ function addQuantity(value, caller) {
     );
 }
 
+$(".item_quantity").focusout(e => {
+    toggleDeleteButton($(e.target))
+})
+
+
+function toggleDeleteButton(target) {
+    console.log(target)
+    let targetVal = $(target).siblings(".item_quantity").addBack().val()
+    console.log(targetVal)
+    if (targetVal > 0) {
+        console.log($(target).siblings(".deleteRow"))
+        $(target).siblings(".deleteRow").hide()
+    }
+    else {
+        $(target).siblings(".deleteRow").show()
+    }
+}
+
+
+$(".item_quantity").each(function (index, element) {
```

Comment:
![image](https://user-images.githubusercontent.com/19233251/80317002-2cee0d80-87cf-11ea-9e67-6710754907a7.png)

Reply:
> this works, try it out. and document.onLoad doesn't work properly for modals

copy and pasted the reply from the original comment